apps/system/js/statusbar.js

```diff
@@ -451,23 +468,32 @@ var StatusBar = {
   },
 
   _paused: 0,
+
   pauseUpdate: function sb_pauseUpdate() {
     this._paused++;
+    this.log('pauseUpdate()', `_paused = ${this._paused}`);
   },
 
   resumeUpdate: function sb_resumeUpdate() {
     this._paused--;
+    this.log('resumeUpdate()', `_paused = ${this._paused}`);
     this._updateIconVisibility();
   },
 
   isPaused: function sb_isPaused() {
+    this.log('isPaused()', `isPaused() = ${(this._paused > 0)}`);
+
+    // @todo Change status bar style here to emphasize the state of !== 0.
```

Comment:
great idea! maybe just adding an gray outline when the debug is on and the statusbar is paused